doc/articles/create-an-app-rider.md

```diff
@@ -67,6 +67,15 @@ Select the Skia.WPF project, then Run.
 
 ***
 
-## Explore
+## Next Steps
 
-Next, explore [Uno Platform features and architecture](xref:Uno.GetStarted.Explore).
\ No newline at end of file
+Now that you're Created and Debug the App.
+
+Learn more about:
```

Comment:
Maybe something like: "Now that you've created and debugged the App, learn more about:"

Reply:
Made in PR https://github.com/unoplatform/uno/pull/14774